myway-api/src/main/java/com/hmel/myway/controllers/DestinationController.java

```diff
@@ -74,7 +74,7 @@
 		logger.info("Get all destination ");
 		List<Destination> res = new ArrayList<>();
 		try {
-			res = iDestinationService.findAll();
+			res = iDestinationService.findAll(0,Integer.MAX_VALUE);
```

Comment:
It is not the best practice to search MAX_VALUE elements, maybe we can cut just first 50-100 elements?